emdeechege/Questioner
# #162951784 Builds application Landing Page

### What does this PR do?
* It adds a landing page for the application
### Description of Task to be completed?
* Creates welcome page
* Create join meetup button
* Create categories for meetups
### How should this be manually tested?
* GIT clone or fork the repository
* Check into the folder created
* Using your favorite browser open index.html
### Screenshots
![screenshot from 2019-01-05 17-07-58](https://user-images.githubusercontent.com/40524026/50725325-1ec53b00-110d-11e9-81c1-de092545dee9.png)
![screenshot from 2019-01-05 17-08-11](https://user-images.githubusercontent.com/40524026/50725326-1ec53b00-110d-11e9-8739-314470a6c6a2.png)


### What are the relevant pivotal tracker stories?
#162951784
